src/axolotl/utils/schemas/peft.py

```diff
@@ -128,6 +128,16 @@ class LoraConfig(BaseModel):
             "description": "loraplus learning rate for lora embedding layers. Default value is 1e-6."
         },
     )
+    use_qalora: bool | None = Field(
+        default=False,
+        json_schema_extra={
+            "description": "Enable Quantization-Aware Low-Rank Adaptation"
+        },
+    )
+    qalora_group_size: int | None = Field(
+        default=16,
```

Comment:
We may want default to be None, otherwise config will auto set this.
```suggestion
        default=None,
```